createdisk.sh

```diff
@@ -370,7 +376,7 @@ mkdir $libvirtDestDir
 
 create_qemu_image $libvirtDestDir
 
-copy_additional_files $1 $libvirtDestDir
+copy_additional_files $1 $libvirtDestDir $internalIP
```

Comment:
we just use global variables for the other variables in the metadata file, so it's not strictly required to explicity pass it here.